modules/post-install-edge-aws-extend-machineset.adoc

```diff
@@ -0,0 +1,51 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/aws-compute-edge-tasks.adoc
+
+:_content-type: PROCEDURE
+[id="post-install-edge-aws-extend-machineset"]
+= Creating a machine set manifest for AWS Local Zones node
+
+After you created subnets in an AWS Local Zones, you can create a machine set manifest.
```

Comment:
```suggestion
After you create subnets in an AWS Local Zones, you can create a machine set manifest.
```

Reply:
I was thinking, since you use "After" the past-tense "created" isn't necessary.

Reply:
Should it be a prerequisite for this procedure that you must have subnets in an AWS Local Zone?